test/Yesod/Paginator/WidgetsSpec.hs

```diff
@@ -52,6 +53,27 @@ spec = withApp $ do
             , "</ul>"
             ]
 
+      it "includes all parameters with the same name" $ do
+        get $ SimpleR 10 3 3
+
+        request $ do
+            addGetParam "p" "3"
+            addGetParam "ids[]" "1"
+            addGetParam "ids[]" "2"
```

Comment:
Encoding `[]` makes the test quite messy (see below). For the use case described in #42, the parameters could be named anything as long as they're named the same, but maybe it's worth using `ids[]` since that seems to be a common naming scheme for arrays of values.